### PR TITLE
Handle exam exports when the user only has rights to multiple Schools.

### PR DESCRIPTION
**Pull request by wtritch**
This PR fixes an issue with exporting exams when a user only has rights to multiple schools and no districts.

### CODE REVIEW

File: webapp/src/main/webapp/src/app/organization-export/organization/organization.mapper.ts [hunk 1]
```diff
@@ -118,10 +118,10 @@ export class OrganizationMapper {
     organizations.schools.forEach(school => {
       let node = root;
       if (!Utils.isUndefined(school.districtId)) {
-        node = node.getOrCreate(x => x.id === school.districtId, organizations.districtsById.get(school.districtId))
+        node = node.getOrCreate(x => x.id === school.districtId, this.getDistrict(organizations, school.districtId));
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/organization-export/organization/organization.mapper.ts]: The problem was that when executing the "matcher" `x => x.id === school.districtId` x was null, leading to an NPE when de-referencing `x.id`
I changed the call to create a shallow matchable District and SchoolGroup placeholder that just contains the ID if the district or school group can't be found in the user's organizations.

File: webapp/src/main/webapp/src/app/organization-export/organization/organization.mapper.ts [hunk 2]
```diff
@@ -132,6 +132,26 @@ export class OrganizationMapper {
     return Utils.isUndefined(a) ? b : a;
   }
 
+  private getDistrict(organizations: UserOrganizations, id: number): District {
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/organization-export/organization/organization.mapper.ts]: `getDistrictOrCreatePlaceholder()`

File: webapp/src/main/webapp/src/app/organization-export/organization/organization.mapper.ts [hunk 3]
```diff
@@ -132,6 +132,26 @@ export class OrganizationMapper {
...
+    if (organizations.districtsById.has(id)) {
+      return organizations.districtsById.get(id);
+    }
+
+    let district: District = new District();
```
Comment (esotrope) [inline webapp/src/main/webapp/src/app/organization-export/organization/organization.mapper.ts]: `const`

Comment (esotrope) [reply webapp/src/main/webapp/src/app/organization-export/organization/organization.mapper.ts]: look i'm the @malaffoon of typescript : j

File: webapp/src/main/webapp/src/app/organization-export/organization/organization.mapper.ts (hunk 2 above)
Comment (wtritch) [reply webapp/src/main/webapp/src/app/organization-export/organization/organization.mapper.ts]: Ugh, i’m going to break these into multiple lines with that rename

File: webapp/src/main/webapp/src/app/organization-export/organization/organization.mapper.ts (hunk 3 above)
Comment (wtritch) [reply webapp/src/main/webapp/src/app/organization-export/organization/organization.mapper.ts]: done

File: webapp/src/main/webapp/src/app/organization-export/organization/organization.mapper.ts (hunk 2 above)
Comment (wtritch) [reply webapp/src/main/webapp/src/app/organization-export/organization/organization.mapper.ts]: done

**Pull request merged**